bitbots_body_behavior/config/body_behavior.yaml

```diff
@@ -138,7 +143,7 @@ behavior:
     ball_reaproach_dist: 1.0
 
     # Distance at which the ball is normally approached
-    ball_approach_dist: 0.2
+    ball_approach_dist: 0.25
```

Comment:
Why is this param changed? This may break things significantly, because it is out of the kick area. This results in the behavior from the game against Starkit. Also, why is this necessary? Is the new ball position less accurate and we, therefore, need more distance to avoid it while kicking?

Reply:
ill change it back